etc/spack/defaults/packages.yaml

```diff
@@ -14,10 +14,16 @@
 #   ~/.spack/packages.yaml
 # -------------------------------------------------------------------------
 packages:
+  ibm-mpi:
```

Comment:
As IBM uses our own mpi (binary package, spectrum mpi), we thought this would be the best way to do it. Otherwise, a user has to manually modify/add packages.yaml. We are also learning about the spack, so please let us know what would be the best way.